b_server.md

```diff
@@ -15,245 +19,372 @@ If the server is responding returns:
-  "salesforce": "boolean whether org has a Salesforce connection"
+  "signupToken": "eV7i1ldl8HoeVFBDtL9paUbS2kFl3rlB",
 }
-
 ```
 
-Returns a session_id for the new org admin, and the org parameters.
+Returns the email associated with the signup token.
 
+_Response:_
 ```json
 {
  "success": true,
- "sessionID": "5e1668d7fffaadabf0474088e...",
- "organization": "organization details..."
+ "email": "neworg@example.com"
 }
 ```
 
-{% include starsep.html %}
+Once the signup token has been validated, the organization can be created.
 
-### Checking whether an org admin username is already in use
+_Endpoint:_
+```
+POST http://b-api.jlinc.test/api/complete_signup
+```
 
-_GET https://sandbox.b.jlinclabs.net/api/orgs/exists/{name-to-check}_
+Creates an organization, organization admin, and stores a session.
+
+_Post body:_
+```json
+{
+  "name": "Bob Co",
+  "email": "neworg@example.com",
+  "password": "passwordbird",
+  "passwordConfirmation": "passwordbird",
+  "signupToken": "eV7i1ldl8HoeVFBDtL9paUbS2kFl3rlB",
+  "salesforce": true, // whether or not the organization has a Salesforce connection
+  "keepMeLoggedIn": true
+}
+```
 
-If the name (after lowercasing) is available (i.e. does not exist), the reply result will be `false`. If the name does exist the reply will be `true`.
+Returns a session ID and initial organization details.
 
+_Response:_
 ```json
 {
  "success": true,
- "result": false
+ "sessionId": "5b2dbdc40a22ecc8ca2e12c3da0f0319",
+ "organization": {
+    "apikey": "bobco",
+    "logo": null,
+    "icon": null,
+    "name": "Bob Co",
+    "dpo_email": "neworg@example.com",
+    "domain": null,
+    "contact_phone": null,
+    "contact_phone_2": null,
+    "address": null,
+    "city": null,
+    "post_code": null,
+    "country": null,
+    "communication_channels": {},
+    "requested_data": {},
+    "consents": {},
+    "salesforce": false,
+    "description": null,
+    "state": null,
+    "public": false,
+    "banner": null,
+    "upsell_description": null,
+    "public_allowed": false,
+    "published": false,
+    "marketplace_tags": [],
+    "industries": [],
+    "brands": [],
+    "public_key": "Vi6djLwwoiga_XWE93sdbYPbgP2jxjCjHDNfq5qFcRg",
+    "dcust_id": "Vi6djLwwoiga_XWE93sdbYPbgP2jxjCjHDNfq5qFcRg",
+    "api_secret": "457b855bbfb36a3bd5df045066f34de40b2db65314cef8b9d83ae272f40fc127",
+    "vendor_name": "bobco"
+  }
 }
 ```
-{% include starsep.html %}
 
-### Adding an org admin to an existing org
+{% include starsep.html %}
 
-_POST https://sandbox.b.jlinclabs.net/api/orgs/register_
+<a name="login"></a>
+### Logging in as an organization admin
 
-Requires a signup token created out-of-band
+_Endpoint:_
+```
+POST http://b-api.jlinc.test/api/login
+```
 
+_Post body:_
 ```json
 {
-  "email": "newadmin@example.com",
-  "password": "foobar",
-  "password_confirmation": "foobar",
-  "signuptoken": "5e1668d7fffaadabf0474088e..."
+ "email": "neworg@example.com",
+ "password": "passwordbird",
```

Comment:
omg __DO NOT__ put `passwordbird` in these docks 🤦‍♂️

Reply:
lol